integration-tests/src/omnipool_liquidity_mining.rs

```diff
@@ -538,6 +538,202 @@ fn withdraw_shares_should_work_when_deposit_exists() {
 	});
 }
 
+#[test]
+fn withdraw_shares_should_should_return_reward_to_user_when_bigger_than_ed() {
```

Comment:
Typo in the  name `...should_should...` also I would use "send" instead of "return" but  that's detail .
I don't think this test is correct if I get the name correctly.
`Charlie` already have HDX balance so this test passes without ED lowering on line 547.

Reply:
fixed typos.

Yeah it was indeed not correct the thing was that in transfer the ED for HDX is retrieved from pallet balances, not from asset registry, so this test was indeed incorrect.

Inreased the yield reward percentage, not it should be fine.